examples/cpp-debug-workspace/a.cpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    cout << "hello world!" << endl;
```

Comment:
It would be good to enhance this little test program a bit. e.g. add a few local variables that change value in a loop, so that we can have something to look-for in the memory view. As well at least one function being called so we can test the run-controls a bit (step-in, step-over). 

We can add a multi-threaded example later.